src/utils/aadAdministrativeUnit.ts

```diff
@@ -0,0 +1,30 @@
+import { AdministrativeUnit } from "@microsoft/microsoft-graph-types";
+import { odata } from "./odata.js";
+import { formatting } from "./formatting.js";
+import { Cli } from "../cli/Cli.js";
+
+const graphResource = 'https://graph.microsoft.com';
+
+export const aadAdministrativeUnit = {
+  /**
+   * Get id of an administrative unit by its display name.
+   * @param displayName Administrative unit display name.
+   * @returns Id of the administrative unit.
+   * @throws Error when administrative unit was not found.
+   */
+  async getAdministrativeUnitIdByDisplayName(displayName: string): Promise<string> {
```

Comment:
Let's return the entire administrative unit instead of only its id. That way we can use it for other purposes as well.

Reply:
Changed to return the entire administrative unit.